lldb/source/Plugins/SymbolFile/DWARF/DebugNamesDWARFIndex.cpp

```diff
@@ -527,7 +527,7 @@ void DebugNamesDWARFIndex::GetTypesWithQuery(
   ConstString name = query.GetTypeBasename();
   std::vector<lldb_private::CompilerContext> query_context =
       query.GetContextRef();
-  if (query_context.size() <= 1)
+  if (query_context.size() <= 1 && !query.GetExactMatch())
```

Comment:
Nit (possibly for a separate PR if you prefer): could we change this to `== 1`? Would we ever get an empty context here? That feels like an invalid query. `GetTypes` already unconditionally accesses the zeroth element, which makes me think we're missing a check/assert for an empty context somewhere

Reply:
I think you're looking at the wrong overload. This calls the variant taking a ConstString (line 461). That one doesn't take array so there's no first element to access.

Reply:
Ah whoops, you're right I was looking at the wrong overload.